doc/config.yml

```diff
@@ -59,26 +89,51 @@ Extensions:
         executable: ${ROOT_DIR}
         remove:
             framework: !include ${MOOSE_DIR}/framework/doc/remove.yml
+            cardinal:
+                - /ICs/BulkEnergyConservationIC
+                - /Cardinal/ICs/BulkEnergyConservation/BulkEnergyConservationICAction
+                - /Cardinal/ICs/VolumetricHeatSource/VolumetricHeatSourceICAction
     MooseDocs.extensions.common:
         shortcuts: !include ${MOOSE_DIR}/framework/doc/globals.yml
     MooseDocs.extensions.acronym:
         acronyms:
             framework: !include ${MOOSE_DIR}/framework/doc/acronyms.yml
+            cardinal: !include ${ROOT_DIR}/cardinal/doc/acronyms.yml
             fenix:
                 FENIX: Fusion ENergy Integrated multiphys-X
                 PIC: particle-in-cell
                 CFD: computational fluid dynamics
                 CAD: computer-aided design
     MooseDocs.extensions.sqa:
         active: true
-        categories:
+        categories: # commented-out categories are not available
             fenix: !include ${ROOT_DIR}/doc/sqa_fenix.yml
             framework: !include ${MOOSE_DIR}/framework/doc/sqa_framework.yml
-            ray_tracing: !include ${MOOSE_DIR}/modules/ray_tracing/doc/sqa_ray_tracing.yml
```

Comment:
Why are you removing ray tracing?

Reply:
Oh sorry, I just saw that it was there twice. Good catch!

Reply:
It wasn't there twice - I just moved it.